### ISSUE TITLE
Invalid "main" entry in package.json?

### ISSUE THREAD

**Issue by vladh** [Alex1990/tiny-cookie#13]
"main" in package.json seems to point to `lib/tiny-cookie.js`, but there is no such file. Is there some reasoning behind this? To get the library to work with browserify, I added the following to my package.json:

```javascript
  "browser": {
    "tiny-cookie": "tiny-cookie/dist/tiny-cookie.js"
  },
```

**Comment by Alex1990** [comment]
Sorry for this fault. It has beed fixed https://github.com/Alex1990/tiny-cookie/commit/676c9f976e804cb276d4f06b8a261a1b8429e9b1.

**Comment by vladh** [comment]
No worries, thank you!

**Comment by mogelbrod** [comment]
Will you be releasing the fix soon? :)

**Comment by Alex1990** [comment]
@mogelbrod  it has been released in v2.0.1

**Comment by mogelbrod** [comment]
@Alex1990: I must've been blind when checking the releases tab - my bad! Thanks for the quick reply and a great library!
